scripts/inference.py

```diff
@@ -77,7 +88,16 @@
     dist.init_process_group()
 
 print("loading model")
-model = load_fms_llama(args.model_path)
+distr_param = "tp" if args.distributed else None
```

Comment:
In the existing code, not args.distributed and torch.cuda.device_count() > 1 and WORLD_SIZE=1 this would be the equivalent of `mp` not None